CHANGELOG

```diff
@@ -1,6 +1,29 @@
 devel
 -----
 
+* Added a new subquery optimization called `splice-subqueries`.
+
+	This optimisation splices the execution of subqueries inside the
+	execution of their surrounding query. For subqueries with a fairly
+	small number of returned rows this saves a significant amount of
+	overhead initializing cursors.
+
+	This optimisation is performed by the optimizer rule
+	`splice-subqueries`, and currently only works on suitable
+	subqueries; A subquery becomes unsuitable if it contains a LIMIT,
+	REMOTE, GATHER or a COLLECT node where the operation is not COUNT.
```

Comment:
> or a COLLECT node where the operation is not COUNT

It's the other way round, `COLLECT` with `COUNT` makes the query unsuitable, the other variants do not.

Reply:
This needs to be updated everywhere  (documentation, release notes) as I copy and pasted it from one master version.